<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>upup</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_up">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>增加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_donw">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>减少</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>60</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="slider_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinbox_1"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>71</y>
      <width>239</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QGroupBox" name="direction_group_body">
       <property name="title">
        <string>方向</string>
       </property>
       <widget class="QRadioButton" name="direction_up">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>139</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>向上</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="direction_down">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>139</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>向下</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>时间 ：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSpinBox" name="time_value"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="button_start">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="button_pause">
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="button_stop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <widget class="QMenu" name="menu_Language">
     <property name="title">
      <string>语言(Language)</string>
     </property>
     <addaction name="L_zh"/>
     <addaction name="L_en"/>
    </widget>
    <addaction name="menu_Language"/>
    <addaction name="menu_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="help_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="menu_exit">
   <property name="icon">
    <iconset resource="ico_pic.qrc">
     <normaloff>:/ico/exit_ico.png</normaloff>:/ico/exit_ico.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="help_about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="L_zh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="L_en">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ico_pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>slider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
